lib/service/service.go

```diff
@@ -6080,3 +6108,29 @@ func makeXForwardedForMiddleware(cfg *servicecfg.Config) utils.HTTPMiddleware {
 	}
 	return utils.NoopHTTPMiddleware
 }
+
+func (process *TeleportProcess) newExternalCloudAuditConfigurator() (*externalcloudaudit.Configurator, error) {
+	watcher, err := local.NewClusterExternalAuditWatcher(process.GracefulExitContext(), local.ClusterExternalCloudAuditWatcherConfig{
+		Backend: process.backend,
+		OnChange: func() {
+			// On change of cluster external cloud audit, trigger teleport
+			// reload, because s3 uploader and athena components don't support
+			// live changes to their configuration.
+			process.BroadcastEvent(Event{Name: TeleportReloadEvent})
+		},
+	})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	// Wait for the watcher to init to avoid a race in case the external cloud
```

Comment:
```suggestion
	// Wait for the watcher init to avoid a race in case the external cloud
```
I think you meant watcher init only?